cron_jobs/cron.ts

```diff
@@ -54,6 +55,14 @@ const jobs = [
     isActive: config.featureAutoAccept,
     name: 'Auto accept psychologist from designated universities in DS',
   },
+  {
+    cronTime: '0 */6 * * *', // every 6 hours
+    onTick: cronPsychologists.reactivatePsychologists,
```

Comment:
Refacto idea: we could have one file per cron. (Not to be done here, maybe we can create a card about that)

Reply:
https://trello.com/c/wTG5JiAS/107-use-separate-file-for-each-kind-of-cron-job